<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopCommunication</class>
 <widget class="QWidget" name="ShopCommunication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2010</width>
    <height>1060</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>与客户聊天</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* General UI Styles */
QWidget {
    background-color: #f0f8ff; /* 天蓝色 */
    border-radius: 12px; /* 更圆润的外观 */
}

QScrollArea {
    background-color: white;
    border-radius: 12px;
    box-shadow: 0px 2px 10px rgba(0, 0, 0, 0.1); /* 微妙阴影效果 */
}

QListWidget {
    background-color: white;
    border: 1px solid #87cefa; /* 浅天蓝色 */
    border-radius: 12px;
    font-size: 16px;
    color: #333;
    box-shadow: 0px 2px 5px rgba(0, 0, 0, 0.05); /* 轻微阴影效果 */
    transition: all 0.3s ease; /* 平滑过渡效果 */
}

QListWidget::item {
    padding: 12px 16px;
    margin: 4px 0;
    border-radius: 8px;
}

QListWidget::item:hover {
    background-color: #e6f7ff; /* 更柔和的蓝色 */
    transform: scale(1.03); /* 放大效果 */
}

QListWidget::item:selected {
    background-color: #87cefa; /* 浅天蓝色选中 */
    color: white;
    transform: scale(1.05); /* 选中时放大效果 */
}

/* Label Styles */
QLabel {
    font-size: 20px;
    color: #333;
    font-weight: bold;
    padding: 10px;
}

/* Button Styles */
QPushButton {
    background-color: #87cefa; /* 天蓝色 */
    border: 2px solid #87cefa;
    border-radius: 20px;
    padding: 12px 24px;
    font-size: 18px;
    color: white;
    font-weight: bold;
    transition: all 0.3s ease;
    box-shadow: 0px 4px 10px rgba(0, 0, 0, 0.1); /* 更大的阴影效果 */
}

QPushButton:hover {
    background-color: #b0e0e6; /* 浅蓝色 hover */
    transform: translateY(-4px); /* 向上移动效果 */
}

QPushButton:pressed {
    background-color: #4682b4; /* 钢蓝色 */
    transform: translateY(2px); /* 按下时下移效果 */
}

/* Emoji Button Styles */
QPushButton#emojiButton {
    background-color: #ffffff;
    border: 2px solid #87cefa;
    border-radius: 50%;
    padding: 12px;
    font-size: 24px;
    color: #87cefa;
    transition: all 0.3s ease;
    box-shadow: 0px 4px 8px rgba(0, 0, 0, 0.1);
}

QPushButton#emojiButton:hover {
    background-color: #e6f7ff;
    transform: scale(1.1);
}

/* Text Edit Styles */
QTextEdit {
    font-size: 16px;
    padding: 12px;
    border: 1px solid #87cefa;
    border-radius: 12px;
    background-color: white;
    color: #333;
    transition: all 0.3s ease;
}

QTextEdit:focus {
    border: 1px solid #4682b4; /* 钢蓝色 focus */
    box-shadow: 0px 0px 8px rgba(70, 130, 180, 0.5); /* 聚焦时的阴影 */
}

/* Scroll Area Styles */
QScrollBar:vertical, QScrollBar:horizontal {
    background-color: #f0f8ff;
    width: 12px;
    border-radius: 6px;
}

QScrollBar::handle:vertical, QScrollBar::handle:horizontal {
    background-color: #87cefa;
    border-radius: 6px;
}

QScrollBar::handle:vertical:hover, QScrollBar::handle:horizontal:hover {
    background-color: #4682b4; /* 钢蓝色 hover */
}
</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <item>
    <widget class="QListWidget" name="customerList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="chatLayout">
     <item>
      <widget class="QLabel" name="customerNameLabel">
       <property name="text">
        <string>客户名称：张三</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="messageArea" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="emojiButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>😊</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="messageInputLayout">
       <item>
        <widget class="QTextEdit" name="messageInputBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>300</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>请输入消息...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
